metadata-ingestion/cli-ingestion.md

```diff
@@ -0,0 +1,94 @@
+# CLI Ingestion
+
+## Ingesting Metadata
+
+```shell
+pip install 'acryl-datahub[datahub-rest]'  # install the required plugin
+datahub ingest -c <path_to_recipe_file>
```

Comment:
let's also link out to the alternate install methods documented here https://docs-website-ie0wod5ko-acryldata.vercel.app/docs/next/cli#alternate-installation-options